analyses/molecular-subtyping-HGG/run-molecular-subtyping-HGG.sh

```diff
@@ -70,14 +76,15 @@ Rscript -e "rmarkdown::render('06-HGG-molecular-subtyping-gene-expression.Rmd',
 
 Rscript -e "rmarkdown::render('07-HGG-molecular-subtyping-combine-table.Rmd', clean = TRUE)"
 
-#### 1p/19q co-deleted oligodendrogliomas notebook -----------------------------
-
-Rscript -e "rmarkdown::render('08-1p19q-codeleted-oligodendrogliomas.Rmd', clean = TRUE)"
+#### Notebooks that are not directly used in subtyping -------------------------
 
-#### HGAT with `BRAF V600E` mutations clustering ------------------------------
+if [ "$SUBSETTING_ONLY" -eq "0" ]; then
+  # 1p/19q co-deleted oligodendrogliomas notebook
+  Rscript -e "rmarkdown::render('08-1p19q-codeleted-oligodendrogliomas.Rmd', clean = TRUE)"
```

Comment:
I guess it seems to me that this module could _in theory_ be used for subtyping? It's hard-coded to old releases with a big TODO note to update.

Reply:
Unlikely that it will get used for subtyping in this project – we didn't find any tumors that met the criteria (and we wouldn't necessarily expect to).